src/components/ListItem.jsx

```diff
@@ -29,6 +29,13 @@ export function ListItem({ listPath, item }) {
 		purchaseItem();
 	};
 
+	const deleteHandler = async (e) => {
+		// Note: Should we add more user feedback when items are successfully deleted? Some might further interrupt usability.
+		if (window.confirm(`Are you sure you'd like to delete ${item.name}?`)) {
```

Comment:
praise: I like how you display the item name in the confirmation dialog, makes it easier to confirm you're deleting the right thing!